### PR TITLE
Change Log Lists update (after chrons update)

### PR DESCRIPTION
**Pull request by Jaygriff**

### CODE REVIEW

**Comment by Nebyab** [comment]
As a heads up, Carbonhell isn't a GM, and as I'm a headmin, might as well remove me from the GM list.

File: html/changelog.html [hunk 1]
```diff
@@ -35,11 +35,10 @@
...
+			<font size='2'><b>Game Masters:</b> Brawzki, Lord_Kruor, Solar Marine, Nebyab, TeamStab, Hutchen, DrunkenFlyingDemoman, Nalar, CarbonHell<br></font>
 			<font size='2'><b>Game Admins:</b> Vievioora, Karmahunter, cvbmn309, Chronitonity, Mocheeze, Doobiedoo23, xYxStarkxYx, Foos, DerptheStewpidGoat, Snake2512, Redswandir, Pyko, Kokojo, WrathofMe9, AlphaDaGreat, AkimAlexstrov, JustAnotherPlayerBro, TDcoolguy300, Lawoggamer, Cobyb, Lcass123<br></font>
-			<font size='2'><b>Coding/Spriting/Mapping Team:</b> Nalar, Redswandir, Mocheeze, Kokojo, Whitefox08, DerptheStewpidGoat, JayGriff, HITHISISFRONK, Dan Hibiki, Nienhaus<br></font>
-			<font size='2'><b>Mentors:</b> McHatters, CarbonHell, Cairo283, Cafaling, Jackrip, Nullbear, NeoPlague, Kathorn, Bawws, Dungeonbones, 8000roxas1234, Deatht1, Voltz6787, DarrelHerring, RG4, JamesButts, ZachOrem, xYxStarkxYx, Markoboy, Zomboing, Rain7x<br></font>
-
+			<font size='2'><b>Coding/Spriting/Mapping Team:</b> Nalar, Redswandir, Mocheeze, Kokojo, Whitefox08, DerptheStewpidGoat, JayGriff, HITHISISFRONK, Dan Hibiki, Nienhaus, 1Block<br></font>
```
Comment (AliiTea) [inline html/changelog.html]: I am a coder and mentor....

**Comment by AliiTea** [comment]
I enjoy being cucked by my dad looping

**Comment by Chronitonity** [comment]
what a weird comment nex

**Comment by Jaygriff** [comment]
fuck you github

**Comment by AliiTea** [comment]
I WIN!

**Pull request closed without merging**